pyproject.toml

```diff
@@ -25,7 +19,8 @@ dependencies = [
     "appdirs",
     "pandas",  # remove!!!
     "python-dateutil",
-    "universal_pathlib"
+    "universal_pathlib",
+    "supabase",
```

Comment:
Could you please pin supabase==1.0.3 as we did in the hub? We ran into things breaking because we hadn't pinned it at some point.